### DISCUSSION TITLE
Start date of task 2 is completed date of task 1

### DISCUSSION THREAD

**Discussion by tiptruck** [TechbeeAT/jtxBoard#888] [General]
Add a task and then add a heap of sub-tasks because they are all part of the same project.  Sub-task 2 can't start until sub-task 1 is completed. So the start date of sub-task 2 should be added when sub-task 1 is completed.

Is it possible to have an option "start date is completion date of task X"?  Do you think it could be useful for others?

Cheers Craig

**Comment by bepolymathe** [comment]
Hi, 

Just my 3 cts.
I understand the idea and the context of use, which is resolving a project in successive steps. However, I think there are also many users who have a more flexible approach, where sub-tasks are more in parallel in term of scheduled/due dates. 

If something like this were envisaged, it should be an option. However, it would not be a standard feature for synchronization with desktop applications such as Thunderbird or evolution.

**Comment by patrickunterwegs** [comment]
Hey, thanks for the discussion!

I do understand that this could be an interesting feature from project management perspective. 
But the problem is that I'm not aware that the standard would offer such an option. This would require the ability to set dependencies that could go in multiple directions (as project management tools do), like start in parallel, start when the other ends, ends when the other starts...

Even if I implement it, it would only work in jtx Board. If then an entry is edited in another app (e.g. Nextcloud web), this could just mess with the functionality and I'm exposed to complaints that the app doesn't work as expected...

So I believe that there are currently more downsides than advantages for such a feature.

**Comment by tiptruck** [comment]
Fair enough. I put it in discussion because I thought it was blurring the line between task management and project management.

Thanks for all the good work

Regards Craig

--

Craig Tipping,

When you stop learning, stop listening, stop looking and asking questions, always new questions, then it is time to die. - Lillian Eugenia Smith

------- Original Message -------
